packages/client/tests/functional/relationMode/tests_m-to-n.ts

```diff
@@ -1087,7 +1085,7 @@ testMatrix.setupTestSuite(
             },
           )
 
-          testIf(isRelationMode_prismaAndSetNull).failing(
+          testIf(isRelationMode_prismaAndSetNull)(
```

Comment:
Do we still need these duplicated tests? or can we remove them if they error like the test with same name but `!isRelationMode_prismaAndSetNull` condition?

Reply:
That should be done now.